openhantek/src/glscope.cpp

```diff
@@ -196,14 +196,15 @@ void GlScope::initializeGL() {
           }
     )";
     const char *fshaderES = R"(
-          #version 100
+          #version 120
```

Comment:
Doesn't make sense this way :/ Open GL ES shader versions are listed here: https://github.com/mattdesl/lwjgl-basics/wiki/glsl-versions#glsl-es-versions-android-ios-webgl. There is no "120".